### PR TITLE
Features/misc improvements

### PR DESCRIPTION
**Pull request by Mikanebu**
* removed `bot adapter` 
* removed extra type in `bot issue` 
* added new regex that matches the following sentence `bot issue "title" in "myorg/myrepo"`
* refactored `bot issue` response to get exact url of issue
* added response section for bot tags `+todo, +link, +standup`
* refactored `bot close milestone` to use same sytax for "title" 
* refactored `bot create|close milestone` to get exact URL
* updated `README.md`
* updated help message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,23 +36,27 @@ Examples:
...
 
 Example:
 ```
-bot issue "This is test issue" about "We want to test our bot create  command" in "datahq/docs"
+bot issue "title of the issue" about "body" in "datahq/docs"
+bot issue "title of the issue" in "datahq/docs"
 ```
 ## Tags
 
+Bot can record channel specified in `config.json` and record into specified Google Spreadsheet, Gist
```
Comment (zelima) [inline README.md]: I do not get this sentence. Think some commas are missing.

**Review by zelima — APPROVED** [review]
Seems fine. Just a small command for README

File: README.md (hunk 1 above)
Comment (Mikanebu) [reply README.md]: @zelima How about `Bot can listen to a specific channel and record it in specified Google Spreadsheet, Gist, configured in config.json`

Comment (zelima) [reply README.md]: much better. how about 
`Bot can listen to a predefined channels ...`

**Pull request merged**